rpc/src/client/transport/http.rs

```diff
@@ -22,6 +22,9 @@ use super::auth;
 
 const USER_AGENT: &str = concat!("tendermint.rs/", env!("CARGO_PKG_VERSION"));
 
+/// `reqwest` http client re-exported for compatibility convenience
+pub type ReqwestClient = reqwest::Client;
```

Comment:
Should we call it `InnerClient` to avoid renaming it if a different lib is used in the future?

```suggestion
pub type InnerClient = reqwest::Client;
```